### DISCUSSION TITLE
Unable to setup AutoPilot Profile

### DISCUSSION THREAD

**Discussion by sunnygaat** [KelvinTegelaar/CIPP#1807] [Q&A]
Failed to add profile for xyzCIPP.onmicrosoft.com: { "_version": 3, "Message": "An error has occurred - Operation ID (for customer support): 00000000-0000-0000-0000-000000000000 - Activity ID: e499c291-87bd-46a1-85c4-d355aeac964a - Url: https://fef.msuc03.manage.microsoft.com/DeviceEnrollmentFE_2309/StatelessDeviceEnrollmentFEService/deviceManagement/windowsAutopilotDeploymentProfiles?api-version=5023-06-28", "CustomApiErrorPhrase": "", "RetryAfter": null, "ErrorSourceService": "", "HttpHeaders": "{}" }

**Comment by KelvinTegelaar** [comment]
This is not the place for support.
